kratos/python/add_modeler_to_python.cpp

```diff
@@ -97,6 +98,12 @@ void  AddModelerToPython(pybind11::module& m)
     py::class_< DuplicateMeshModeler, DuplicateMeshModeler::Pointer, Modeler >(m,"DuplicateMeshModeler")
         .def(py::init<ModelPart&>())
     ;
+
+    py::class_< CopyPropertiesModeler, CopyPropertiesModeler::Pointer, Modeler >(m,"CopyPropertiesModeler")
+        .def(py::init< >())
```

Comment:
I think the default constructor exposure is useless in this case.

Reply:
done